sdk/src/main/java/io/opentelemetry/sdk/trace/config/TraceConfig.java

```diff
@@ -82,8 +82,7 @@
 @Immutable
 public abstract class TraceConfig {
   // These values are the default values for all the global parameters.
-  // TODO: decide which default sampler to use
-  private static final Sampler DEFAULT_SAMPLER = Samplers.alwaysOn();
+  private static final Sampler DEFAULT_SAMPLER = Samplers.parentOrElse(Samplers.alwaysOn());
```

Comment:
I know I am annoying, but can we do this in a followup PR? This is an important change and would like to make sure everyone aggress on, just adding the parentElse is trivial to be merged.

Reply:
That's totally fine! I hadn't realized initially that the default was already set in the spec - I can also create an issue in the spec repo to discuss this instead of doing it in a followup PR here if that makes more sense?

Reply:
I added an issue to the spec repo to discuss this: https://github.com/open-telemetry/opentelemetry-specification/issues/728 cc @anuraaga 

One thing that I admit to being confused about is how the sampler is meant to interact with the `TraceFlags`, which is slightly inconsistent across languages. If anyone can clear up some of the inconsistencies for me that would be helpful.